lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -1687,4 +1692,32 @@ private function generateIdentifierName($identifier)
 
         return strtoupper(dechex(crc32($identifier->getName())));
     }
+
+    protected function getCommentOnTableSQL(string $tableName, ?string $comment) : string
+    {
+        return sprintf(
+            'EXEC sys.sp_addextendedproperty @name=N\'MS_Description\', 
+@value=N%s , @level0type=N\'SCHEMA\',@level0name=N\'dbo\', 
+@level1type=N\'TABLE\',@level1name=N%s',
```

Comment:
How about using nowdoc here, instead of escaping all this single quotes?

Reply:
Done :+1: